components/uuf-httpconnector-msf4j/src/main/java/org/wso2/carbon/uuf/httpconnector/msf4j/MicroserviceHttpRequest.java

```diff
@@ -234,6 +238,11 @@ public int getRemotePort() {
     }
 
     @Override
+    public boolean isGet() {
```

Comment:
Let's move this method to be after `getMethod()` method. Similar to the `HttpRequest` interface.